blog/_posts/2021-08-20-Mitchell-Johnson_Jamie-Schmidt.md

```diff
@@ -0,0 +1,296 @@
+---
+layout: post
+date: 2021-07-31
+author: Mitchell Johnson & Jamie Schmidt
+---
+
+### Development of MABE2
+
+## Introduction
+
+To quote the [Github for MABE2](https://github.com/mercere99/MABE2), MABE is a software framework deigned to easily build and customize software for evolutionary computation or artificial life.  The resulting systems should be useful for studying evolutionary dynamics, solving complex problems, comparing evolving systems, or exploring the open-ended power of evolution.  
+
+MABE, accessible [here](https://github.com/Hintzelab/MABE/wiki), stands for Modular Agent-Based Evolution and was one of the first true digital evolution frameworks.  It was structured in this fashion:
+
+![MABE structure](https://github.com/Hintzelab/MABE/wiki/images/MABE_Overview.png)
+
+and, though powerful, could of course be improved.  The creation of the 2.0 version of MABE, called MABE2, was the focus of our project.  Our main focus was on refining the underlying architecture of MABE to be more end user-friendly, particularly in designing custom experiments.  The culmination of our efforts was in the creation of DynamicOrg, an Organism which was near-completely abstracted to allow for any combination of Genomes and Brains, as assigned through the config file.
```

Comment:
change "and, though" to "Though" since the start of that sentence is separated by the image and this could be a stand alone sentence

Reply:
bold "The culmination..." since that's the main point of all your work and that sentence gets a little lost in the paragrpah

Reply:
good suggestions, done!